internal/healthcheck/healthcheck.go

```diff
@@ -82,12 +80,6 @@ func HealthChecks(ctx context.Context, c client.Reader, l logr.Logger) error {
 	}
 
 	if clusterOperatorsReady && clusterServiceVersionsReady {
-		// Only check SriovNetworkNodeState once cluster operators and CSVs are stable
-		if err := IsSriovNetworkNodeReady(ctx, c, l); err != nil {
```

Comment:
We still need the SRIOV network node ready health check. It's just the workaround in the upgrade_handlers.go that can be removed

Reply:
oh gotcha, I've shrunk the scope of the PR